docs/assessResources.md

```diff
@@ -0,0 +1,42 @@
+
+# Procedure to assess resources used by Observability Operator
+
+1. Provision an OpenShift cluster
+
+2. Run `oc apply -f hack/olm/catalog-src.yaml` to install the Observability Operator (OO) catalogue.
+
+3. Using the UI install OO
+
+4. Scale down the following deployments, so we can remove the currently set limits on OO:
+
+```bash
+# Scale down the cluster version operator
+oc -n openshift-cluster-version scale deployment.apps/cluster-version-operator --replicas=0
```

Comment:
should we scale down the CVO? Can't we edit the cluster-version to put OLM in unmanaged mode? 🤔